package.json

```diff
@@ -40,13 +42,17 @@
     "url": "https://github.com/panderalabs/koa-pageable/issues"
   },
   "files": [
```

Comment:
This will also publish the entire types folder alongside the built lib

Reply:
So I just thought of something, you may just want to specify the `types/index.d.ts` file.  I don't think we should be including the config and test files with the library.